integration/src/test/java/org/hibernate/validator/integration/wildfly/jpa/CustomValidatorFactoryInPersistenceUnitIT.java

```diff
@@ -86,6 +89,31 @@ public void testValidatorFactoryPassedToPersistenceUnitIsCorrectlyConfigured() t
 		log.debug( "testValidatorFactoryPassedToPersistenceUnitIsCorrectlyConfigured completed" );
 	}
 
+	/**
+	 * Makes sure the HV added via our module ZIP is used instead the one coming with WF by relying on functionality not
+	 * present in the WF-provided HV
+	 */
+	// TODO How to make that work reliably also after a HV upgrade within WF?
```

Comment:
So I suppose we will need a new feature unsupported in WildFly on each Validator upgrade in WildFly?

There's no way to get and check the module version used by WildFly?